stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -144,6 +143,7 @@ public struct Job: Sendable {
   }
 }
 
+#if !SWIFT_STDLIB_TASK_TO_THREAD_MODEL_CONCURRENCY
```

Comment:
I mostly just exposed this type to ensure it will work. I was hoping @ktoso will make sure the guards / changes needed to expose the whole API involving the type is correct. I can drop this commit from the PR so you can do that.

Reply:
I'll just drop this change then. it was mainly for CI testing purposes. I'll let you re-enable the type and its associated API's completely.